app/views/claims/support/claims/_filter.html.erb

```diff
@@ -0,0 +1,153 @@
+<div class="app-filter-layout">
+  <div class="app-filter-layout__filter">
+    <div class="app-filter__header">
+      <div class="app-filter__header-title">
+        <h2 class="govuk-heading-m"><%= t("filter") %></h2>
+      </div>
+      <div class="app-filter__header-action"></div>
+    </div>
+    <div class="app-filter-layout__content">
+      <% if @filter_form.filters_selected? %>
+        <div class="app-filter-layout__selected">
+          <div class="app-filter__selected-heading">
+            <div class="app-filter__heading-title">
+              <h2 class="govuk-heading-m govuk-!-margin-bottom-1"><%= t("selected_filters") %></h2>
+              <p class="govuk-body">
+                <%= govuk_link_to(
+                  t("clear_filters"),
+                  @filter_form.clear_filters_path,
+                  no_visited_state: true,
+                ) %>
+              </p>
+            </div>
+          </div>
+
+          <% if @filter_form.school_ids.present? %>
+            <h3 class="govuk-heading-s govuk-!-margin-bottom-0"><%= t("claims.support.claims.index.school") %></h3>
+            <ul class="app-filter-tags">
+              <% @filter_form.school_ids.each do |school_id| %>
+                <li>
+                 <%= govuk_link_to(
+                   Claims::School.find_by_id(school_id)&.name,
+                   @filter_form.index_path_without_filter(filter: "school_ids", value: school_id),
+                   class: "app-filter__tag",
+                   no_visited_state: true,
+                 ) %>
+                </li>
+              <% end %>
+            </ul>
+          <% end %>
+
+           <% if @filter_form.provider_ids.present? %>
+             <h3 class="govuk-heading-s govuk-!-margin-bottom-0"><%= t("claims.support.claims.index.accredited_provider") %></h3>
+             <ul class="app-filter-tags">
+               <% @filter_form.provider_ids.each do |provider_id| %>
+                 <li>
+                  <%= govuk_link_to(
+                    Claims::Provider.find_by_id(provider_id)&.name,
```

Comment:
We should avoid calling database queries in views.

Let's add a `#providers` method to the `filter_form` that loads all providers by `provider_ids` list. This will also prevent N+1 queries.

```ruby
class Claims::ClaimQuery
  def providers
    Claims::Provider.find(provider_ids)
  end
end
```

Reply:
`filter_form.providers` here too. :)